tcti/tcti_device.c

```diff
@@ -54,17 +54,11 @@ TSS2_RC LocalTpmSendTpmCommand(
     UINT32 commandCode;
     UINT32 cnt;
 #endif
-    printf_type rmPrefix;
 
     rval = tcti_send_checks (tctiContext, command_buffer);
     if (rval != TSS2_RC_SUCCESS) {
         return rval;
     }
-    if (tcti_intel->status.rmDebugPrefix == 1) {
-        rmPrefix = RM_PREFIX;
-    } else {
-        rmPrefix = NO_PREFIX;
-    }
 #ifdef DEBUG
```

Comment:
We should not need these #ifdef DEBUG anymore

Reply:
Done